packages/data-models/deployment.model.ts

```diff
@@ -146,9 +146,11 @@ interface IContentsEntry {
   relativePath: string;
   size_kb: number;
   modifiedTime: string;
-  md5Checksum: string;
+  md5Checksum?: string;
```

Comment:
nit(blocking)
This interface has a note that it duplicates the interface used in scripts, however the changes added makes this no longer apply.
To avoid confusion it would be better to create a new interface `IAssetContentsEntry` that extends.

Additionally would you be able to confirm why md5Checksum may now no longer be included? Seems very useful to have